nservicebus/hosting/azure-functions-service-bus/isolated-worker.md

```diff
@@ -67,6 +67,8 @@ snippet: asb-function-isolated-configure-error-queue
 
 The configuration API exposes NServiceBus transport configuration options via the `configuration.Transport` property to allow customization. However, not all of the options will be applicable to execution within Azure Functions.
```

Comment:
Suggestion: 
However, not all options will be applicable to execution within Azure Functions.

Reply:
Applied here https://github.com/Particular/docs.particular.net/commit/445a89d6badfa65fea4ef61d48af5e609828586e